include/babeltrace/debuginfo.h

```diff
@@ -0,0 +1,29 @@
+#ifndef _BABELTRACE_DEBUGINFO_H
+#define _BABELTRACE_DEBUGINFO_H
+
+#include <stdint.h>
+#include <stdbool.h>
+#include <stddef.h>
+
+struct debug_info_source {
+	/* Strings are owned by this */
+	char *func;
+	long unsigned int line_no;
```

Comment:
I'll have to double check but the 64-bit requirement was from the DWARF spec and that's what `libdw` returns. I'll make the switch to `uint64_t`, though.